### PR TITLE
docs: HOSTS command

### PR DESCRIPTION
**Pull request by alexcb**
Signed-off-by: Alex Couture-Beil <alex@earthly.dev>

### CODE REVIEW

File: docs/earthfile/earthfile.md [hunk 1]
```diff
@@ -1371,6 +1371,16 @@ The `USER` command sets the user name (or UID) and optionally the user group (or
 
 The `WORKDIR` command `strs` the working directory for other commands that follow in the recipe. The working directory is also persisted as the default directory for the image. If the directory does not exist, it is automatically created. This command works the same way as the [Dockerfile `WORKDIR` command](https://docs.docker.com/engine/reference/builder/#workdir).
```
Comment (alexcb) [inline docs/earthfile/earthfile.md:1372]: offtopic: what is `strs`?

Comment (vladaionescu) [reply docs/earthfile/earthfile.md:1372]: Not sure...?

File: docs/earthfile/earthfile.md [hunk 2]
```diff
@@ -1371,6 +1371,16 @@ The `USER` command sets the user name (or UID) and optionally the user group (or
...
 
+## HOST
```
Comment (vladaionescu) [inline docs/earthfile/earthfile.md:1374]: We should say that this command is experimental and needs to be enabled with a feature flag.

File: docs/earthfile/earthfile.md (hunk 1 above)
Comment (alexcb) [reply docs/earthfile/earthfile.md:1372]: here's an update for that: https://github.com/earthly/earthly/pull/1798

File: docs/earthfile/earthfile.md [hunk 3]
```diff
@@ -1400,6 +1400,25 @@ Sets an initialization time period in which failures are not counted towards the
...
+The `HOST` command creates a hostname entry (under `/etc/hosts`) that causes `<hostname>` to resolve to the specified `<ip>` address.
+
+{% hint style='info' %}
+##### Note
+The `HOST` command is experimental and must be enabled by enabling the `--use-host-command` flag, e.g.
+
+```Dockerfile
+VERSION --use-host-command 0.6
+```
+{% endhint %}
```
Comment (vladaionescu) [inline docs/earthfile/earthfile.md:1420]: I would put this part right below the `## HOST` title, and above `### Synopsis`

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**